src/sections/dataset/dataset-summary/DatasetSummary.tsx

```diff
@@ -0,0 +1,17 @@
+import { DatasetField } from '../../../../src/dataset/domain/models/Dataset'
+import { SummaryFields } from './SummaryFields'
+import { License as LicenseModel } from './../../../../src/dataset/domain/models/Dataset'
+import { License } from './License'
+interface DatasetSummaryProps {
+  summaryFields: DatasetField[]
+  license: LicenseModel
+}
+
+export function DatasetSummary({ summaryFields, license }: DatasetSummaryProps) {
+  return summaryFields && license ? (
+    <article>
+      <SummaryFields summaryFields={summaryFields}></SummaryFields>
+      <License license={license}></License>
+    </article>
+  ) : null
+}
```

Comment:
It seems like this props cannot be null so we don't need to check if they exists `{ summaryFields, license }`. We could get rid of `summaryFields && license ?`

I was looking at the Dataset form and the summary fields seems to be required fields so there's always going to be summaryFields. We don't need to add a check for that

The license object seems to be added automatically to any new Datasets so it cannot be null neither.

If there is a situation where they can be null we can add the check and change the props definition, otherwise we don't need to check them

The same for the License and SummaryFields components, if we have a check in this first level we don't need a check in those components

Reply:
ok, thanks, I will remove the check for null, and remove the undefined type from the props

Reply:
Sorry for the noise if I'm misunderstanding, but are you saying that the metadata fields that appear near the top of the dataset page below the blue citation box and above the tabs will never be null or empty?

<kbd><img width="1181" alt="Screenshot 2023-05-16 at 8 02 06 AM" src="https://github.com/IQSS/dataverse-frontend/assets/18374574/7e8b8b27-02b3-47c6-8520-33b16263a369"></kbd>

In that screenshot, keywords appear, but Harvard Dataverse doesn't force depositors to add keywords. And the [customdatasetsummaryfields](https://guides.dataverse.org/en/5.13/installation/config.html?highlight=customdatasetsummaryfields#customdatasetsummaryfields) setting let's any Dataverse installation decide which metadata fields appear in that section.

Reply:
@MellyGray I removed the null check from License and SummaryFields, but when I tried to remove it from DataSummary, the component test failed, saying the map variable within SummaryFields was undefined. 
So I committed the change with the null check removed for License & SummaryFields, let me know what you think

Reply:
I just looked at the code https://github.com/IQSS/dataverse/blob/fc23042ab337dafb1c92f2aab15c73a223bedcec/src/main/java/edu/harvard/iq/dataverse/dataset/DatasetUtil.java#L44 and if custom summary fields is null or empty, it returns a default list of fields, so I think we don't have to worry about that scenario.

Reply:
> @MellyGray I removed the null check from License and SummaryFields, but when I tried to remove it from DataSummary, the component test failed, saying the map variable within SummaryFields was undefined.
> So I committed the change with the null check removed for License & SummaryFields, let me know what you think

I was looking at the code and the error about the` the map variable within SummaryFields was undefined` is because the DatasetMother mock needs to be updated to contain the summaryFields and license. I just did a commit to add those fields to DatasetMother and remove the check from DatasetSummary.tsx

I'm going to approve this, thanks!